pkg/controller/account/account_controller.go

```diff
@@ -325,7 +325,8 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 
 	if (currentAcctInstance.Status.State == "") && (currentAcctInstance.Status.Claimed == false) {
 		// before doing anything make sure we are not over the limit if we are just error
-		accountTotal, err := TotalAwsAccounts(awsSetupClient)
+		reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
```

Comment:
Why did we add this again its already declared here: https://github.com/openshift/aws-account-operator/blob/041603a044a4238565ab79cc60ccb96082ae00ea/pkg/controller/account/account_controller.go#L236

Reply:
+1 this line should be removed. Probably a mistake.